i18n/fr/docusaurus-plugin-content-docs/version-22.04/releases/centreon-core.md

```diff
@@ -16,6 +16,37 @@ notre [Github](https://github.com/centreon/centreon/issues/new/choose).
 
 ## Centreon Web
 
+### 22.04.6
+
+#### Bug fixes
+
+- [Authentication] Fixed user theme retrieval via OpenID Connect login
```

Comment:
[Core] Improved database storage to avoid blocking Broker when maximum values are reached